types/zen-push/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for zen-push 0.1
+// Project: https://github.com/zenparsing/zen-push
+// Definitions by: daprahamian <https://github.com/daprahamian>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as Observable from 'zen-observable';
+
+declare class PushStream<T> {
+    readonly observable: Observable<T>;
+    readonly observed: number;
+    next(x: T): void;
+    error(e: Error): void;
+    complete(x?: any): void;
+}
+
+declare namespace PushStream {}
```

Comment:
the readme for zen-push says to `import PushStream from 'zen-push'`, which means that you should drop the `export = PushStream` and this namespace hack and instead `export default class PushStream<T> ...`.

You might actually need to import it as `import PushStream = require('zen-push')` in which case you should keep the `export = PushStream` but get rid of the namespace.

Reply:
My issue is that the module does not export as ES6 module, but instead uses a `module.exports = `. My thinking was that using an `export =` would allow the bundle to work for people who did not have `allowSyntheticDefaultExports` enabled.

If that is not the case, then I will gladly change it to an `export default`